packages/environment/scripts/deploy-pendle-oracles/decrypt-keystore.ts

```diff
@@ -0,0 +1,23 @@
+import * as fs from "node:fs";
+import { Wallet } from "ethers";
+import inquirer from "inquirer";
+
+export async function decryptKeystore({
+  keystoreFilePath,
+}: {
+  keystoreFilePath: string;
+}) {
+  const { password } = await inquirer.prompt([
+    {
+      type: "password",
+      name: "password",
+      message: "Enter your password:",
+      mask: "",
+    },
+  ]);
+
+  const json = fs.readFileSync(keystoreFilePath, "utf8");
+  const wallet = await Wallet.fromEncryptedJson(json, password);
```

Comment:
Can we do this without ethers? The deployer is a non-privileged account, so we can either save it as plain-text, or use a simple encryption/decryption mechanism, like `bcrypt`, or